docs/index.md

```diff
@@ -30,7 +30,8 @@ PM > Install-Package Arcus.WebApi.Logging
 
 The `Arcus.WebApi.Security` package contains functionality to easily add security capabilities to an API.
 
-- [Shared access key authentication](features/shared-access-key.md)
```

Comment:
Careful, I did some changes to this one recently

Reply:
Ok, just merged with remote upstream. No conflicts.

Reply:
Awesome, thanks!